src/train_model.py

```diff
@@ -505,8 +548,8 @@ def train_and_save(network, optimizer, criterion, train_loader,
             loss = get_loss(network, data, criterion, dataset, device)
             if (cluster_gradient
                     and i % cluster_gradient_config['frequency'] == 0):
-                if cluster_gradient_config['normalize']:
-                    normalize_weights(network)
+                # if cluster_gradient_config['normalize']:
```

Comment:
why are these commented out?

Reply:
No idea, I think that was a mistake. Sorry and thanks for flagging!